packages/database/src/migrations/20210211224521-AddConfigToProjects-modifies-schema.js

```diff
@@ -0,0 +1,37 @@
+'use strict';
+
+var dbm;
+var type;
+var seed;
+
+/**
+ * We receive the dbmigrate dependency from dbmigrate initially.
+ * This enables us to not have to rely on NODE_PATH.
+ */
+exports.setup = function (options, seedLink) {
+  dbm = options.dbmigrate;
+  type = dbm.dataType;
+  seed = seedLink;
+};
+
+/**
+ *
+ *
+ * @param db
+ * @returns {{replacementParams: *, query: *}}
+ */
+exports.up = function (db) {
+  db.runSql(`ALTER TABLE project DROP COLUMN IF EXISTS tile_sets;`);
```

Comment:
Do we need to migrate the old `tile_sets` data across to `config`, for `laos_eoc`? Seems safer than relying on us remembering to add it back in through the admin panel